roles/pcp/tasks/pmie.yml

```
@@ -53,10 +53,10 @@
   loop: "{{ __pcp_global_webhook_action_status.results }}"
   changed_when:
     - pcp_pmie_endpoint | length > 0
-    - item.found == 0
+    - item.found is defined and item.found == 0
```

```suggestion
    - item.found | d(-1) == 0
```
the `d` (short for `defined`) is also a filter which returns the default value if the item tested is not defined - you can use some other value other than `-1` to mean "not found"

---

Thanks @richm . Suggestion has been applied.